compiler/hash-reporting/src/report.rs

```diff
@@ -252,10 +218,11 @@ impl ReportCodeBlock {
                 index + 1,
                 pad_width = longest_indent_width
             );
+
             writeln!(
                 f,
                 "{} {}   {}",
-                if (start_row..=end_row).contains(&index) {
+                if (start_row..=end_row).contains(&index) && !line.is_empty() {
```

Comment:
Will this not skip over some lines in the file?

Reply:
No since this is also occurring on the print below as well